src/view/reviewManager.ts

```diff
@@ -525,7 +577,7 @@ export class ReviewManager implements vscode.DecorationProvider {
 					}
 
 					const oldComments = commentsForFile[fileName];
-					const uri = vscode.Uri.parse(nodePath.join(`commit~${commit.substr(0, 8)}`, fileName));
+					const uri = vscode.Uri.file(nodePath.join(`commit~${commit.substr(0, 8)}`, fileName));
```

Comment:
Which of these methods should we use here, file or parse?

Reply:
`file` is correct

Reply:
should be file:

```js
	const uri = vscode.Uri.file(nodePath.join(`commit~${commit.substr(0, 8)}`, fileName));
```

Reply:
```js
const uri = vscode.Uri.file(nodePath.join(`commit~${commit.substr(0, 8)}`, fileName));
```

is correct

Reply:
```js
const uri = vscode.Uri.file(nodePath.join(`commit~${commit.substr(0, 8)}`, fileName));
```
is correct

Reply:
```js
const uri = vscode.Uri.file(nodePath.join(`commit~${commit.substr(0, 8)}`, fileName));
```
is correct

Reply:
```js
const uri = vscode.Uri.file(nodePath.join(`commit~${commit.substr(0, 8)}`, fileName));
```
is correct

Reply:
```js
const uri = vscode.Uri.file(nodePath.join(`commit~${commit.substr(0, 8)}`, fileName));
```
is correct!

Reply:
```js
const uri = vscode.Uri.file(nodePath.join(`commit~${commit.substr(0, 8)}`, fileName));
```
is correct